packages/flutter_test/lib/src/test_default_binary_messenger.dart

```diff
@@ -302,6 +311,82 @@ class TestDefaultBinaryMessenger extends BinaryMessenger {
     }, handler);
   }
 
+  /// Set a handler for intercepting stream events sent to the
+  /// platform on the given channel.
+  ///
+  /// Intercepted method calls are not forwarded to the platform.
+  ///
+  /// The given handler will replace the currently registered
+  /// handler for that channel, if any. To stop intercepting messages
+  /// at all, pass null as the handler.
+  ///
+  /// Events are decoded using the codec of the channel.
+  ///
+  /// The handler's stream messages are used as a response, after re-encoding
+  /// them using the channel's codec.
+  ///
+  /// To send an error, send a [PlatformException] in the handler.
```

Comment:
How does one send a PlatformException? Or is it supposed to be:

```suggestion
  /// To send an error, throw a [PlatformException] in the handler.
```

Reply:
I must have never updated these docs to match the new code